libsolidity/analysis/OverrideChecker.cpp

```diff
@@ -791,18 +822,28 @@ void OverrideChecker::checkOverrideList(OverrideProxy _item, OverrideProxyBySign
 		missingContracts = expectedContracts - specifiedContracts;
 
 	if (!missingContracts.empty())
+	{
+		vector<ContractDefinition const*> sortedMissingContracts;
+		for (size_t i = _item.contract().annotation().linearizedBaseContracts.size() - 1; i > 0; i--)
+		{
+			auto contract = _item.contract().annotation().linearizedBaseContracts[i];
+			if (missingContracts.count(contract))
+				sortedMissingContracts.emplace_back(contract);
+		}
```

Comment:
```suggestion
for (
	auto const* contract: _item.contract().annotation().linearizedBaseContracts |
	boost::adaptor::filter([](auto const* contract) -> bool { return missingContracts.count(contract); }
)
  sortedMissingContracts.emplace_back(contract);
```
?

Reply:
Your suggestion is no longer reversing the linearized list?